install/source/chapter10.rst

```diff
@@ -12,7 +12,7 @@ the web portal as the Master Administrator.
 
```

Comment:
let's change the very first sentence to get rid of all those clients...
You can brand web portal, Windows client, Mac client, iOS client and Android client.

could be something like this?
You can brand web portal, Windows and Mac clients, iOS and Android Apps.